src/App.js

```diff
@@ -65,13 +76,12 @@ function App() {
   useEffect(() => {
     const cachedCoordinates = getLocalStorage('1km_user_coordinates');
     if (cachedCoordinates) {
-      dispatch({ type: 'setUserCoordinates', payload: cachedCoordinates });
-      dispatch({ type: 'setModalState', payload: false });
+      dispatch({ type: 'setInitialData', payload: cachedCoordinates });
     }
   }, []);
 
   useEffect(() => {
-    async function fetchProtests() {
+    async function fetchProtests({ onlyMarkers = false } = {}) {
```

Comment:
can you add a comment here explaning what `onlyMarkers` is for?

Reply:
done